### ISSUE TITLE
Hypoid Gears

### ISSUE THREAD

**Issue by dedani** [meadiode/cq_gears#11]
Hi, is there any intend to add a generator for hypoid gears?

**Comment by meadiode** [comment]
Yes I have intentions to add hypoid gears also. Though, haven't done enough research on them, but I will get to it.
